### DISCUSSION TITLE
Trying to build old project but not working

### DISCUSSION THREAD

**Discussion by Vanheden** [npm/rfcs#773] [Question]
Hello I'm trying to build this https://github.com/ERR1R/rartracker but I get this. I have no idea what to do with NPm to be honest.

> 0 info it worked if it ends with ok
> 1 verbose cli [ '/usr/bin/node', '/usr/local/bin/npm', 'install' ]
> 2 info using npm@3.5.2
> 3 info using node@v18.19.1
> 4 verbose stack ReferenceError: primordials is not defined
> 4 verbose stack     at evalmachine.<anonymous>:43:5
> 4 verbose stack     at Object.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/cmd-shim/node_modules/graceful->
> 4 verbose stack     at Module._compile (node:internal/modules/cjs/loader:1356:14)
> 4 verbose stack     at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
> 4 verbose stack     at Module.load (node:internal/modules/cjs/loader:1197:32)
> 4 verbose stack     at Module._load (node:internal/modules/cjs/loader:1013:12)
> 4 verbose stack     at Module.require (node:internal/modules/cjs/loader:1225:19)
> 4 verbose stack     at require (node:internal/modules/helpers:177:18)
> 4 verbose stack     at Object.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/cmd-shim/node_modules/graceful->
> 4 verbose stack     at Module._compile (node:internal/modules/cjs/loader:1356:14)
> 5 verbose cwd /var/www/rartracker
> 6 error Linux 5.15.146.1-microsoft-standard-WSL2
> 7 error argv "/usr/bin/node" "/usr/local/bin/npm" "install"
> 8 error node v18.19.1
> 9 error npm  v3.5.2
> 10 error primordials is not defined
> 11 error If you need help, you may report this error at:
> 11 error     <https://github.com/npm/npm/issues>
> 12 verbose exit [ 1, true ]

**Comment by ljharb** [comment]
You're using an ancient version of npm with an almost EOL version of node. you might want to reinstall node so you get the npm version that comes with it.

Either way, further questions should go to something like Stack Overflow; this discussion forum is for change requests for the npm cli.
